specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2618 @@
+              "$ref": "#/definitions/amlFilesystemNamespaceHydrationStatus"
+            },
+            "archiveSettings" : {
+              "description": "Archive settings and status",
+              "$ref" :  "#/definitions/amlFilesystemArchive"
+            }
+          },
+          "type": "object"
+        },
+        "sku": {
+          "description": "SKU for the amlFilesystem",
+          "properties": {
+            "name": {
+              "description": "SKU name for this amlFilesystem",
+              "type": "string"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "type": "object"
+    },
+    "amlFilesystemName" : {
+      "description": "Name for the amlFilesystem.  Alphanumerics, underscores, and hyphens. Start and end with alphanumeric.",
+      "type" : "string",
+      "pattern": "^[0-9a-zA-Z][-0-9a-zA-Z_]{0,78}[0-9a-zA-Z]$",
+      "minLength": 2,
+      "maxLength": 80
+    },
+    "amlFilesystemsListResult": {
+      "description": "Result of the request to list Caches. It contains a list of Caches and a URL link to get the next set of results.",
+      "properties": {
+        "nextLink": {
+          "description": "URL to get the next set of Cache list results, if there are any.",
+          "type": "string"
+        },
+        "value": {
+          "description": "List of Caches.",
+          "items": {
+            "$ref": "#/definitions/amlFilesystem"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "amlFilesystemHealth": {
+      "description": "An indication of amlFilesystem health. Gives more information about health than just that related to provisioning.",
+      "properties": {
+        "state": {
+          "description": "List of amlFilesystem health states.",
+          "enum": [
+            "Unknown",
+            "Unavailable",
+            "Available",
+            "Degraded",
+            "Transitioning",
+            "Maintenance"
+          ],
+          "x-ms-enum": {
+            "name": "amlFilesystemHealthStateType",
+            "modelAsString": true
+          },
+          "type": "string"
+        },
+        "statusDescription": {
+          "description": "Describes of state.",
+          "type": "string"
+        }
+      }
+    },
+    "amlFilesystemEncryptionSettings": {
+      "description": "amlFilesystem encryption settings.",
+      "properties": {
+        "keyEncryptionKey": {
+          "x-ms-secret": true,
+          "$ref": "#/definitions/KeyVaultKeyReference",
+          "description": "Specifies the location of the encryption key in Key Vault."
+        }
+      }
+    },
+    "amlFilesystemHSMSettings": {
+      "description": "amlFilesystem HSM settings.",
+      "properties": {
+        "container": {
+          "description": "Container used for the amlFilesystem.",
+          "$ref": "#/definitions/URLString"
+        },
+        "importPrefix" : {
```

Comment:
The importPrefix is readOnly. I'll let BrainB/Joe weigh in on whether that's also true for exportPrefix.

Reply:
Fixed.
I didn't modify exportPrefix yet.  I'll wait to hear from Brian or Joe.

Reply:
I believe that the exportPrefix is readOnly at the same times that importPrefix is readOnly.

Both are R/W at cluster creation time, and should be readOnly afterwards.

Reply:
Oops, Joe is right; it doesn't matter if it can change after creation; it must be settable at creation time.